src/services/tx/sponsoredCall.ts

```diff
@@ -0,0 +1,35 @@
+import { SAFE_GELATO_RELAY_SERVICE_URL } from '@/config/constants'
+import { type SafeTransactionData } from '@safe-global/safe-core-sdk-types'
+
+// TODO: import type from relay-service
+export type SponsoredCallPayload = {
+  chainId: string
+  to: string
+  data: SafeTransactionData['data'] | undefined
+  gasLimit?: string | number
+}
+
+export const getSafeGelatoRelayServiceUrl = () => {
+  return `${SAFE_GELATO_RELAY_SERVICE_URL}/v1/relay`
```

Comment:
There is no dynamic value in this so it can be imported from the constant directly.